vignettes/fr/datatable-fread-and-fwrite.Rmd

```diff
@@ -0,0 +1,295 @@
+
+### 1.5 Prise en charge de `integer64`
+
+Par défaut, `fread` détecte les entiers plus grands que 2<sup>31</sup> et les lit en tant que `bit64::integer64` pour préserver la précision complète. Ce comportement peut être redéfini de trois manières :
+
+- par colonne : utiliser l'argument `colClasses` pour spécifier le type des colonnes individuelles.
+
+- par appel : utiliser l'argument `integer64` dans `fread()` pour définir la manière dont toutes les colonnes `integer64` détectées seront lues.
+
+- globalement : définir l'option `datatable.integer64` de votre session R ou du fichier `.Rprofile` pour modifier le comportement par défaut pour tous les appels de fread.
+
+L'argument integer64 et l'option correspondante acceptent les valeurs suivantes :
+
+- `"integer64"` (par défaut) : lit les entiers longs en tant que `bit64::integer64` avec la précision complète.
+
+- `"double"` ou `"numeric"`: lit les entiers longs en tant que nombres à double précision, éventuellement en perdant discrètement la précision (similaire à `utils::read.csv` en base R).
+
+- `"character"` : lit les entiers longs en tant que chaînes de caractères.
+
+Pour vérifier ou définir le comportement global par default, utilisez :
+
+```{r}
+# le comportement par défaut de fread est de traiter les entiers longs comme des "integer64"; mais ce paramètre global peut être modifié :
+options(datatable.integer64 = "double")   # Exemple : définir globalement à "double"
+getOption("datatable.integer64") 
+```
+
+### 1.6 Ignorer ou sélectionner les colonnes par nom ou par position
+
+Pour économiser de la mémoire et pour améliorer les performances, utilisez les arguments `select` ou `drop` de `fread()` pour lire uniquement les colonnes dont vous avez besoin.
+
+- Si vous avez besoin uniquement de quelques colonnes, utilisez `select`.
+- Si vous voulez exclure uniquement quelques colonnes, utilisez `drop` — ceci évite de lister tout ce que vous voulez garder.
+
+Points clé:
+- `select` : vecteur des noms des colonnes ou des positions à garder (les autres seront ignorés).
+- `drop`: vecteur des noms des colonnes ou des positions à ignorer (les autres sont gardés).
+- N'utilisez pas `select` et `drop` simultanément — ils sont mutuellement exclusifs.
+- `fread()` vous avertira si une colonne que vous avez spécifiée n'existe pas dans le fichier.
+
+Pour les détails, voir la page du manuel en exécutant `?fread` dans R.
+
+### 1.7 Sauter à une ligne de l'entête d'une sous-table en utilisant une sous-chaîne d'un nom de colonne
+
+Utilisez `skip="chaîne"` dans `fread` pour chercher une ligne contenant une sous-chaîne (typiquement issue des noms de colonne, par exemple `skip="Date"`). La lecture commence à la première ligne où la correspondance est trouvée. Ceci est utile pour sauter les métadonnées ou pour sélectionner des sous-tables dans les fichiers contenant des tables multiples. Cette fonctionnalité est inspirée de la fonction `read.xls` du paquet gdata .
```

Comment:
I did not know about this feature, but I had to read the english version to understand what it does.
can this title be clarified?

Reply:
OK, in https://github.com/Rdatatable/data.table/pull/7253/commits/748631bb9cbf916c5daa1492227cd176edbd578e I deleted that section heading, and moved the corresponding text up to the other section which discusses the skip defaults. Does that make sense to you?